content-src/components/DiscoveryStreamComponents/Navigation/Navigation.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+export class Topic extends React.PureComponent {
+  render() {
+    const {url, name} = this.props;
+    return (<li><a key={name} href={url}>{name}</a></li>);
+  }
+}
+
+export class Navigation extends React.PureComponent {
+  render() {
+    const {links} = this.props || [];
+    const {alignment} = this.props || "centered";
+    const moreRecs = links.pop();
+    return (
+      <span className={`ds-navigation ds-navigation-${alignment}`}>
+        <ul>
+          {links && links.map(t => <Topic key={t.name} url={t.url} name={t.name} />)}
+          {moreRecs ? <li><a href={moreRecs.url} className="ds-more-recommendations">{moreRecs.name}</a></li> : null }
```

Comment:
Seems like the last item is still a `Topic` so maybe have that component handle "more" or "last" prop.

Reply:
Last item is more recs, but handled like a topic as implemented by the server.